scripts/thrift/README.md

```diff
@@ -0,0 +1,41 @@
+# Python client for CodeChecker
+`client.py` contains simple API requests to a CodeChecker server and it
+can be a starting point how you can write your own script.
+
+Before you run this example program you have to do the following steps to
+setup and environment:
+
+```sh
+# Create a Python virtualenv and set it as your environment.
+python3 -m venv venv
+source $PWD/venv/bin/activate
+
+# Install thrift package.
+pip3 install thrift==0.13.0
+
+# Get and install CodeChecker API packages.
+#
+# It will download API packages for the 'v6.19.1' but you can download newer
+# versions as well.
+#
+# WARNING: make sure that the package versions are not newer than what the
+# CodeChecker server uses.
+wget https://github.com/Ericsson/codechecker/raw/v6.19.1/web/api/py/codechecker_api/dist/codechecker_api.tar.gz && \
+pip3 install codechecker_api.tar.gz && \
+rm -rf codechecker_api.tar.gz
+
+wget https://github.com/Ericsson/codechecker/raw/v6.19.1/web/api/py/codechecker_api_shared/dist/codechecker_api_shared.tar.gz && \
+pip3 install codechecker_api_shared.tar.gz && \
+rm -rf codechecker_api_shared.tar.gz
```

Comment:
Are these packages not available from PIP?

Reply:
No, these are available only in the repository as local python packages. This way open source contributors are also able to change the API easily and there is no need extra permissions for pypi and npmjs.